components/molecules/BenefitTasks.js

```diff
@@ -21,7 +21,7 @@ export default function BenefitTasks(props) {
               <a href={value.taskLink}>
                 <img
                   src={value.taskIcon}
-                  className="font-normal pb-3 px-8"
+                  className="font-normal pb-3 px-4"
```

Comment:
the `p` tag is using a `px-5` should this not be the same to match it?

Reply:
Fixed